### DISCUSSION TITLE
Alex's Caves

### DISCUSSION THREAD

**Discussion by Elnhomantia** [DynamicTreesTeam/DynamicTrees#931] [Compatibility Addon Suggestions]
Version 1.20.1
Link : https://www.curseforge.com/minecraft/mc-mods/alexs-caves

2 trees in the primordial caves and 1 in the forlorn hollows

**Comment by szunakabeats** [comment]
1.20.1 please :)

**Comment by szunakabeats** [comment]
is this still active ? :)
